### DISCUSSION TITLE
Translation of select and multiselect fields

### DISCUSSION THREAD

**Discussion by filippo-groppi** [pimcore/pimcore#17919] [Q&A / Support]
I would like to translate the values ​​of the options of a select and ensure that there are no discrepancies between the selected values ​​in the various languages.

1 - I thought of creating a select field in a localized field - Result: the inserted value could differ in the different languages.
2 - I thought of using the Data Objects Select Options - Result: here it does not let me insert the translations
3 - I thought of using one to many and many to many relational fields that connect to objects with a localized field with the value - Result: complex to manage especially if we then go to import the selected values, perhaps inside an Object Brick. Also the dilemma: create a class for each option (but then when I import I have to be careful) or create a single generic class for the options but with the risk of having hundreds of thousands of objects of the class.

Better ideas?

Ty

**Comment by idna001** [comment]
Hey, 
I am not sure if there is a better Pimcore way besides localized fields. I'm actually using a customer specific solution in this case: I have my own class for translations, where I can just create and edit symfony translations. 
So I have values for selected fields e. g. like 'option.select.one' and translate it into multiple languages via the data object. 

What also works quite well is the DeepL plugin that does translations automatically by saving your object. 

It's a bit of a workaround, but it works well. 

Hope this helps, Andi

**Comment by dpfahlbusch** [comment]
You can use translation keys in the display value field and translate them via admin translations
